services/GroupService_test.go

```diff
@@ -123,7 +136,7 @@ var _ = ginkgo.Describe("Group", func() {
 			ginkgo.BeforeEach(func() {
 				recName = groupDelete
 
-				group = resources.CreateGroupWithID(existingGroupID)
+				group = resources.CreateGroupWithID(126)
```

Comment:
Come on, did you really think I will miss these numbers? :smile: You have `existingGroupID` and `nonExistingGroupID` constants, use those.